app.js

```diff
@@ -0,0 +1,31 @@
+const dropdownTrigger = document.querySelector('.dropdown-parent');
+const dropdownMenu = document.querySelector('.dropdown-menu');
+const navLinks = document.getElementsByClassName('nav-link');
+
+// const dropdownAction = () => {
+  // const hideMenuHelper = (element) => {
+  //   if (dropdownMenu.style.visibility === 'visible') {
+  //     element.addEventListener('mousever', () => {
+  //       console.log('hovering!');
+  //       dropdownMenu.style.visibility = 'hidden';
+  //     });
+  //   }
+  // }
+
+  const showMenu = (element) => {
```

Comment:
Well done! I'm glad to see you guys got to the JS portion of this assignemnt! It looks good too!